src/main/java/hudson/plugins/tfs/model/PullRequestMergeCommitCreatedEventArgs.java

```diff
@@ -9,6 +9,267 @@
 
 public class PullRequestMergeCommitCreatedEventArgs extends GitCodePushedEventArgs {
+
+    static final String SAMPLE_REQUEST_PAYLOAD =
+        "{\n" +
+        "    \"collectionUri\":\"https://fabrikam-fiber-inc.visualstudio.com\",\n" +
+        "    \"repoUri\":\"https://fabrikam-fiber-inc.visualstudio.com/Personal/_git/olivida.tfs-plugin\",\n" +
+        "    \"projectId\":\"Personal\",\n" +
+        "    \"repoId\":\"olivida.tfs-plugin\",\n" +
+        "    \"commit\":\"6a23fc7afec31f0a14bade6544bed4f16492e6d2\",\n" +
+        "    \"pushedBy\":\"olivida\",\n" +
+        "    \"pullRequestId\":42,\n" +
+        "    \"iterationId\":2,\n" +
+        "    \"workItems\":\n" +
+        "    [\n" +
+        "        {\n" +
+        "          \"id\": 297,\n" +
+        "          \"rev\": 1,\n" +
+        "          \"fields\": {\n" +
+        "            \"System.Id\": 297,\n" +
+        "            \"System.AreaId\": 3570,\n" +
+        "            \"System.AreaPath\": \"Fabrikam-Fiber-Git\",\n" +
+        "            \"System.NodeName\": \"Fabrikam-Fiber-Git\",\n" +
+        "            \"System.TeamProject\": \"Fabrikam-Fiber-Git\",\n" +
+        "            \"System.AreaLevel1\": \"Fabrikam-Fiber-Git\",\n" +
+        "            \"System.Rev\": 1,\n" +
+        "            \"System.AuthorizedDate\": \"2014-12-29T20:49:20.77Z\",\n" +
+        "            \"System.RevisedDate\": \"9999-01-01T00:00:00Z\",\n" +
+        "            \"System.IterationId\": 3570,\n" +
+        "            \"System.IterationPath\": \"Fabrikam-Fiber-Git\",\n" +
+        "            \"System.IterationLevel1\": \"Fabrikam-Fiber-Git\",\n" +
+        "            \"System.WorkItemType\": \"Product Backlog Item\",\n" +
+        "            \"System.State\": \"New\",\n" +
+        "            \"System.Reason\": \"New backlog item\",\n" +
+        "            \"System.CreatedDate\": \"2014-12-29T20:49:20.77Z\",\n" +
+        "            \"System.CreatedBy\": \"Jamal Hartnett <fabrikamfiber4@hotmail.com>\",\n" +
+        "            \"System.ChangedDate\": \"2014-12-29T20:49:20.77Z\",\n" +
+        "            \"System.ChangedBy\": \"Jamal Hartnett <fabrikamfiber4@hotmail.com>\",\n" +
+        "            \"System.AuthorizedAs\": \"Jamal Hartnett <fabrikamfiber4@hotmail.com>\",\n" +
+        "            \"System.PersonId\": 77331,\n" +
+        "            \"System.Watermark\": 607,\n" +
+        "            \"System.Title\": \"Customer can sign in using their Microsoft Account\",\n" +
+        "            \"Microsoft.VSTS.Scheduling.Effort\": 8,\n" +
+        "            \"WEF_6CB513B6E70E43499D9FC94E5BBFB784_System.ExtensionMarker\": true,\n" +
+        "            \"WEF_6CB513B6E70E43499D9FC94E5BBFB784_Kanban.Column\": \"New\",\n" +
+        "            \"System.Description\": \"Our authorization logic needs to allow for users with Microsoft accounts (formerly Live Ids) - http://msdn.microsoft.com/en-us/library/live/hh826547.aspx\"\n" +

```

Comment:
Huh, good catch!  I just copy/pasted the _sample response_ from [Get a list of work items > By IDs](https://www.visualstudio.com/en-us/docs/integrate/api/wit/work-items#by-ids).  Can you relay your suggestions to them, too?